app/js/trouble.js

```diff
@@ -1,22 +1,33 @@
-'use strict';
-
 (function (gbl, $) {
+  
+  'use strict';
 	
 	gbl.dropdown = function (options) {
 		
-		var $dropdown = options.dropdown,
-			$toggle = options.toggleButton,
-			token = +new Date(),
-			ns = options.namespace
-			elList = $('.dropdowns'),
-			controlsMegaMenu = (options.controlsMegaMenu && options.controlsMegaMenu == false)? false: true;
-			clickAnywhereToClose = (options.clickAnywhereToClose && options.clickAnywhereToClose == false) ? false : true,
-			transitionEnd = gbl.utilities.whichTransitionEvent();
+		var $dropdown = options.dropdown;
+		var	$toggle = options.toggleButton;	
+		var	ns = options.namespace;		
+		var	clickAnywhereToClose;
+
+		if (options.clickAnywhereToClose && options.clickAnywhereToClose === false) {

```

Comment:
Nice - even if it doesn't generate a linting error, rewriting code to be more readable is always a good choice.